CHANGES.md

```diff
@@ -185,6 +185,9 @@ invalid cylc command is used (instead of a ValueError).
 
 ### Documentation
 
+[#xxxx](https://github.com/cylc/cylc-flow/pull/xxxx) - moved documentation to
```

Comment:
Can fill this with 3181.

Reply:
I think this is PR is actually the fix for #3180 ? (it already has the close #... comment in the PR text)

Reply:
Done.

(We put PR links in the change log, not the issues they close ... because sometimes there is no issue that the close ... although this does create an annoying chicken-and-egg situation when first posting a PR - hence the 'xxxx' placeholders!)